demo/web/firebase_element/app_element.html

```diff
@@ -0,0 +1,39 @@
+<dom-module id="app-element">
+  <template>
+    <style>
+      :host {
+        display: block;
+      }
+    </style>
+
+    <div class="horizontal center-justified layout">
+      <div class="vertical-section">
+        <h3>Dinosaurs by Height</h3>
+        <template is="dom-bind">
+          <firebase-collection
+              order-by-child="height"
+              location="https://dinosaur-facts.firebaseio.com/dinosaurs"
+              data="{{dinosaurs}}"></firebase-collection>
+          <template is="dom-repeat" items="[[dinosaurs]]" as="dinosaur">
```

Comment:
Yup, but some styles are not applied when I wrap the demo inside an `<app-element>`.

To import styles in the entry page you use `<style is="custom-style"...>` in the imported file, to import the styles in an element the styles need to be wrapped in `<dom-element id="demo-pages"><template>`.
See also: https://www.polymer-project.org/1.0/docs/devguide/styling.html#style-modules
Some styles in `demo-pages.html` as it currently is, are only applied on the entry page and only target elements in the entry page, but not elements inside other elements.
I modified `polymer_elements/lib/src/paper-styles/demo-pages.html` locally from `<style is="custom-style">` to `<dom-module id="demo-pages><template>` and the styles are applied inside `<app-element>`.

Creating a copy of the about 30 lines of styles in `demo-pages.html` file wrapped as `<dom-module id="demo-pages">` (inside `demo-pages.html` or in a new file) would fix this.
What do you think?